.travis.yml

```diff
@@ -4,8 +4,23 @@ go:
   - 1.8
 
 dist: trusty
-sudo: false
+sudo: required
 
+services:
+  - docker
+cache:
+  directories:
+    - "$HOME/google-cloud-sdk/"
+env:
+  - DOCKER_COMPOSE_VERSION="1.13.0" PATH=$PATH:${HOME}/google-cloud-sdk/bin CLOUDSDK_CORE_DISABLE_PROMPTS=1 PROJECT_NAME=transparent-keyserver
```

Comment:
If you split them in several lines/items one build will be triggered per line: https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml
But it seems
```
env:
  global:
     # separate line per VAR
``` 
does what we want. Will do.